go/engine/pgp_import_key_test.go

```diff
@@ -351,7 +351,7 @@ func numPrivateGPGKeys(g *libkb.GlobalContext) (int, error) {
 	return index.Len(), nil
 }
 
-func armorKey(t *testing.T, tc libkb.TestContext, email string) (libkb.PGPFingerprint, keybase1.KID, string) {
+func genPGPKeyAndArmor(t *testing.T, tc libkb.TestContext, email string) (libkb.PGPFingerprint, keybase1.KID, string) {
```

Comment:
Renamed `armorKey` to `genPGPKeyAndArmor` to make it clear that it generates a new key each call.